tests/util/block.ts

```diff
@@ -185,12 +185,9 @@ export const verifyBlockFees = async (
 
             // We are only interested in fee paying extrinsics:
             // Either ethereum transactions or signed extrinsics with fees (substrate tx)
-            // TODO: sudo should not have paysFee
-            if (
-              dispatchInfo.paysFee.isYes &&
```

Comment:
See comments above in this PR for more more context. My understanding is that:

* Ethereum transactions should be `pays_fee: No` when successful
* Substrate transactions should be `pays_fee: Yes` with exceptions like sudo
* Inherents should be `pays_fee: No`